torch_frame/data/multi_tensor.py

```diff
@@ -55,21 +53,18 @@ def __repr__(self) -> str:
     def shape(self) -> tuple[int, int, int]:
         return (self.num_rows, self.num_cols, -1)
 
-    @property
-    def ndim(self) -> int:
```

Comment:
mypy thinks property is a `Callable` and thus caused an error at different places. Since `ndim` is constant across instances, I have changed it to a class variable.